stubs/playsound/METADATA.toml

```diff
@@ -1,2 +1,7 @@
 version = "1.3.*"
 upstream_repository = "https://github.com/TaylorSMarks/playsound"
+
+[tool.stubtest]
+ignore_missing_stub = true
+# TODO: Does not compile with Python 3.12 as of 2024-10-18 (release 1.3.0).
+skip = true
```

Comment:
What's the issue? It installs for me on Windows and Ubuntu (WSL) 3.12
Comments are a bit all over the place about this: https://github.com/TaylorSMarks/playsound/issues/150

Reply:
Something something PEP 517? (https://github.com/astral-sh/uv/issues/2252#issuecomment-1981973117)

Reply:
Ah right, we use uv in the CI. pip install does fail with `--use-pep517`

Reply:
We actually use `uv` for most of our CI, but we _don't_ use `uv` in CI for third-party stubtest for specifically this reason -- a bunch of packages we have stubs for aren't PEP-517-compliant!

Reply:
@Avasam https://github.com/python/typeshed/actions/runs/11401636351/job/31724969081

```python
      Traceback (most recent call last):
        File "/tmp/stubtest-vubw4ied/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
          main()
        File "/tmp/stubtest-vubw4ied/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/tmp/stubtest-vubw4ied/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
                 ^^^^^^^^^^^^^^^^^^^^^
        File "/tmp/pip-build-env-ov07etnk/overlay/lib/python3.12/site-packages/setuptools/build_meta.py", line 332, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=[])
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/tmp/pip-build-env-ov07etnk/overlay/lib/python3.12/site-packages/setuptools/build_meta.py", line 302, in _get_build_requires
          self.run_setup()
        File "/tmp/pip-build-env-ov07etnk/overlay/lib/python3.12/site-packages/setuptools/build_meta.py", line 516, in run_setup
          super().run_setup(setup_script=setup_script)
        File "/tmp/pip-build-env-ov07etnk/overlay/lib/python3.12/site-packages/setuptools/build_meta.py", line 318, in run_setup
          exec(code, locals())
        File "<string>", line 6, in <module>
        File "/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/inspect.py", line 1285, in getsource
          lines, lnum = getsourcelines(object)
                        ^^^^^^^^^^^^^^^^^^^^^^
        File "/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/inspect.py", line 1267, in getsourcelines
          lines, lnum = findsource(object)
                        ^^^^^^^^^^^^^^^^^^
        File "/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/inspect.py", line 1096, in findsource
          raise OSError('could not get source code')
      OSError: could not get source code
```

Reply:
If we don't use --pep517 here... maybe installing wheel on the CI is sufficient?
Locally:
- `python3.12 -m pip uninstall playsound wheel -y`
- `python3.12 -m pip cache purge` (make sure it doesn't use cached wheel)
- `python3.12 -m pip install playsound`: Gets the error above
- `python3.12 -m pip install wheel`
- `python3.12 -m pip install playsound`: Works

Reply:
Considering that playsound development seems to have stalled and I don't expect huge changes to the stubs, I'd rather remove them than invest time into working around the problems.